FEM/Material/Solid/MohrCoulombFailureCriterion.h

```diff
@@ -0,0 +1,37 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * \file MohrCoulombFailureCriterion.h
+ *
+ * Created on February 6, 2019, 1:40 PM
+ *
+ */
+
+#pragma once
+
+namespace SolidProp
+{
+class MohrCoulombFailureCriterion
```

Comment:
i think WX has already implemented MohrCoulomb stuff. Isn't it possible to reuse it?